### DISCUSSION TITLE
ppstructure的版面分析功能中,det模型和layout模型分别是干什么的？

### DISCUSSION THREAD

**Discussion by qihanghou726** [PaddlePaddle/PaddleOCR#14135] [Documents]
为什么我同一个layout模型，不同的det模型会使同一个东西识别成表格或者图片？

**Comment by qihanghou726** [comment]
英文det+英文layout+英文table。             表格检测完整    图片表格区别差       表格容易检测为图片  
中文det+中文layout+ 中文table。		表格检测不完整    图片表格区别好
中文det+英文layout + 中文table。            表格检测完整    图片表格区别差       图片容易检测为表格
英文det + 中文layout+ 英文table。		表格检测不完整    图片表格区别好
这个是我发现的规律 还想请教一下各位

**Comment by ErnestinaQiu** [comment]
我的理解是
det模型是在图片中检测出文字框区域
layout模型是区域识别，就是将图片的大片文字区域识别为文本区"Text"，标题区域"Title"这种
